app/views/spree/shared/_payment.html.haml

```diff
@@ -13,3 +13,10 @@
     = t :email_payment_summary
 - if @order.payments.any?
   = render partial: 'spree/shared/payments_list', locals: { payments: @order.payments }
+
+- if @order.payments.any?
```

Comment:
we can reuse the if from line 14, right?

Reply:
I'm always confused with indentation and `if` statement in haml file. Will take a look!

Reply:
Addressed via caf246bd8